### ISSUE TITLE
RDS aws_db_instance Apply stuck in loop due to missing performance_insights_kms_key_id when enabling Performance Insights

### ISSUE THREAD

**Issue by brycehemme** [terraform-providers/terraform-provider-aws#9406]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Terraform Version
0.11.7

### Affected Resource(s)
* aws_db_instance

### Terraform Configuration Files
inside of an aws_db_instance resource (note not setting performance_insights_kms_key_id) :

```hcl 
  performance_insights_enabled          = true
  performance_insights_retention_period = 7
```

### Debug Output
These are the params sent to the AWS API that result in a HTTP 500:
```
Action=ModifyDBInstance&ApplyImmediately=false&DBInstanceIdentifier=records-db&EnablePerformanceInsights=true&PerformanceInsightsKMSKeyId=&PerformanceInsightsRetentionPeriod=7&Version=2014-10-31
```

### Expected Behavior
Fail with an error stating that performance_insights_kms_key_id must be provided.

### Actual Behavior
Stuck in a loop of "Still Modifying" DB instance. It should be noted that the actual RDS instance is not in a "modifying" state. When killing the apply it results in the Terraform state having no reference to the aws_db_instance resource any longer.

### Steps to Reproduce
Follow the above details.

**Comment by azhurbilo** [comment]
use for workaround

```
data "aws_kms_alias" "rds" {
  name = "alias/aws/rds"
}

resource "aws_db_instance" "default" {
...
   performance_insights_enabled          = "${var.performance_insights_enabled}"
   performance_insights_kms_key_id       = "${var.performance_insights_enabled ? data.aws_kms_alias.rds.target_key_arn : ""}"
   performance_insights_retention_period = "${var.performance_insights_retention_days_period}"
}
```

**Comment by bflad** [comment]
Fix submitted: https://github.com/terraform-providers/terraform-provider-aws/pull/9745

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 2.24.0 of the Terraform AWS Provider, tomorrow. 👍
